firmware/controllers/bench_test.cpp

```diff
@@ -65,34 +65,27 @@ bool isRunningBenchTest(void) {
 
 static void runBench(brain_pin_e brainPin, OutputPin *output, float delayMs, float onTimeMs, float offTimeMs,
 		int count) {
-    int delaySt = delayMs < 1 ? 1 : TIME_MS2I(delayMs);
-	int onTimeSt = onTimeMs < 1 ? 1 : TIME_MS2I(onTimeMs);
-	int offTimeSt = offTimeMs < 1 ? 1 : TIME_MS2I(offTimeMs);
-	if (delaySt < 0) {
-		scheduleMsg(logger, "Invalid delay %.2f", delayMs);
-		return;
-	}
-	if (onTimeSt <= 0) {
-		scheduleMsg(logger, "Invalid onTime %.2f", onTimeMs);
-		return;
-	}
-	if (offTimeSt <= 0) {
-		scheduleMsg(logger, "Invalid offTime %.2f", offTimeMs);
-		return;
-	}
+	int delayUs = delayMs * 1000;
+	int onTimeUs = onTimeMs * 1000;
+	int offTimeUs = offTimeMs * 1000;
+
+	delayUs = delayUs < 1000 ? 1000 : delayUs;
+	onTimeUs = onTimeUs < 1000 ? 1000 : onTimeUs;
+	offTimeUs = offTimeUs < 1000 ? 1000 : offTimeUs;
+
 	scheduleMsg(logger, "Running bench: ON_TIME=%.2f ms OFF_TIME=%.2fms Counter=%d", onTimeMs, offTimeMs, count);
```

Comment:
we apply thresholds to parameters but print unmodified values here

also it looks like
int delayUs = maxF(1, delayMs) * 1000;
would be a shorter way to express?

to be honest "1000" bothers me as a magic constant I would prefer MS2US here

Reply:
cleaned it up a bit, how's that?